src/components/searchbar/searchbar.ts

```diff
@@ -359,7 +359,8 @@ export class Searchbar extends BaseInput<string> {
   }
 
   setFocus() {
-    this._renderer.invokeElementMethod(this._searchbarInput.nativeElement, 'focus');
+    let onElement = this._renderer.selectRootElement(this._searchbarInput.nativeElement);
```

Comment:
Given that the implementation of [`selectRootElement`](https://github.com/angular/angular/blob/95698d93ad1944ea0c5bad3dea6608b730558b56/packages/platform-browser/src/dom/dom_renderer.ts#L137), what is the benefit that it provides here given that you seemingly already have a reference to the native element?